charts/kubeflow-pipelines/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: kubeflow-pipelines
 description: GetInData ML Ops Platform - Kubeflow Pipelines (Platform Agnostic), adapted from official GCP Helm Chart
 icon: https://getindata.com/img/favicon-32x32.png
 type: application
-version: 1.6.0
+version: 1.6.1
```

Comment:
Do you think it's possible to do a biinary-kernel-like versioning here? So X.Y.Z-BUILDNUM? Like `1.6.0-1`?

Reply:
Added `<version core> "+" <build>` as per https://semver.org/spec/v2.0.0.html

Reply:
Linter does not allow it, reverting.
As SemVer doc says - build is not considered during version comparison.